### PR TITLE
[backport v2.8.x] remove v1 monitoring code (#10539)

### PR DESCRIPTION
**Pull request by codyrancher**
### Summary
Part of these two 'epic' backports:
https://github.com/rancher/dashboard/issues/10276
https://github.com/rancher/dashboard/issues/10277

Backport of https://github.com/rancher/dashboard/issues/10490

The `node-list.spec.ts` and `__tests__/index.test.ts`  do not exist in the 2.8 release branch so I omitted those changes. 

- https://github.com/rancher/dashboard/tree/release-2.8/cypress/e2e/tests/pages/explorer
- https://github.com/rancher/dashboard/tree/release-2.8/shell/pages/c/_cluster/explorer

### CODE REVIEW

File: shell/pages/c/_cluster/explorer/index.vue [hunk 1]
```diff
@@ -178,16 +175,6 @@ export default {
       return this.$store.getters['management/all'](MANAGEMENT.CLUSTER);
     },
 
-    hidePspDeprecationBanner: mapPref(PSP_DEPRECATION_BANNER),
```
Comment (aalves08) [inline shell/pages/c/_cluster/explorer/index.vue:181]: Is this to be deleted in 2.8? It's still used in the code (template part)

File: shell/pages/c/_cluster/explorer/index.vue [hunk 2]
```diff
@@ -566,7 +550,7 @@ export default {
       />
     </div>
 
-    <div v-if="!hasV1Monitoring && componentServices">
+    <div v-if="clusterServices">
```
Comment (aalves08) [inline shell/pages/c/_cluster/explorer/index.vue:553]: should be `componentServices`

**Review by aalves08 — COMMENTED** [review]
Added a couple of comments. Also tested v2 monitoring and it seems to work.

File: shell/pages/c/_cluster/explorer/index.vue (hunk 1 above)
Comment (codyrancher) [reply shell/pages/c/_cluster/explorer/index.vue:181]: Alright, it looks like https://github.com/aalves08/dashboard/commit/92d810399ac7f56aad5fec7cba23ed0e2be1ab28#diff-36dedf9fa870d90a88aecacdcdc91c2c1f0043ce96183e1677c480839d475a4cL30 is what got included. 

I'll go through this pr compared to your other one line by line to make sure there wasn't any other issue. Sorry about that.

File: shell/pages/c/_cluster/explorer/index.vue (hunk 2 above)
Comment (codyrancher) [reply shell/pages/c/_cluster/explorer/index.vue]: Looks like this was the last difference I could spot. Really good eye man. Thanks.

**Comment by codyrancher** [comment]
Only took 5 test reruns 
image

**Review by aalves08 — APPROVED** [review]
LGTM

**Pull request merged**